docs/sources/setup/install/helm/reference.md

```diff
@@ -8005,7 +7912,7 @@ false
 		<tr>
 			<td>monitoring.serviceMonitor.metricsInstance</td>
 			<td>object</td>
-			<td>If defined, will create a MetricsInstance for the Grafana Agent Operator.</td>
+			<td>DEPRECATED If defined, will create a MetricsInstance for the Grafana Agent Operator.</td>
```

Comment:
The Agent Operator is still Deprecated and EOL Nov. 1 of this year.  If we can use this setting with Alloy, then we need to update the docs to say so.

Reply:
This setting creates a grafana-agent CRD object.  There are alternative settings for the prometheus-operator CRD objects which Alloy supports.